src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
+        <fieldset class="fieldInput">
+          <legend class="title1">Personal information</legend>
+          <label class="unit">
+            Surname:
+            <input
+              type="text"
+              id="surname"
+              name="surname"
+              required
+              autocomplete="off"
+              minlength="1"
+              maxlength="30"
+            >
+          </label>
+          <label class="unit">
+              Name:
+            <input
+             type="text"
+             id="fname"
+             name="fname" required
+             autocomplete="off"
+             minlength="1"
+             maxlength="30"
```

Comment:
whenever you have more than 2 attributes you should write them from a new line
```suggestion
             type="text"
             id="fname"
             name="fname" 
             required
             autocomplete="off"
             minlength="1"
             maxlength="30"
```